pymc3/tests/test_distributions_random.py

```diff
@@ -1472,12 +1472,6 @@ def ref_rand(size, mu, sigma, upper):
             pm.TruncatedNormal, {"mu": R, "sigma": Rplusbig, "upper": Rplusbig}, ref_rand=ref_rand
         )
 
-    def test_skew_normal(self):
-        def ref_rand(size, alpha, mu, sigma):
-            return st.skewnorm.rvs(size=size, a=alpha, loc=mu, scale=sigma)
-
-        pymc3_random(pm.SkewNormal, {"mu": R, "sigma": Rplus, "alpha": R}, ref_rand=ref_rand)
-
```

Comment:
why did you remove it?

Reply:
It's one of thosr old random tests we have been replacing with the BaseTestDistribution class. Should have been removed when we refactored it but I must have missed it.

Also, I checked that it was passing locally before removing it. It did fail in this PR but it's not surprising since it is a non-seeded statistical test